artemis-commons/src/main/java/org/apache/activemq/artemis/utils/uri/URISchema.java

```diff
@@ -31,7 +31,7 @@ public T newObject(URI uri, P param) throws Exception {
    }
 
    public void populateObject(URI uri, T bean) throws Exception {
-      BeanSupport.setData(uri, bean, parseQuery(uri.getQuery(), null));
```

Comment:
we have ignored this per design in the past.

- the Protocol manager will try to set properties that will not be available on every protocol, and that will be using the TCP Schema as far as I know.
- Also there's an issue between versions. We store the JNDI with the properties for the connection factory. and it may not have a certain property between versions.


But I am more concerned about the protocol manager. Have you tested this?

Reply:
where does the protocol manager set the properties using the TCP schema?